.codex/skills/ci-check-name-parity/references/comparison-matrix.md

```
@@ -0,0 +1,21 @@
+# CI Check Comparison Matrix
+
+## Table of Contents
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)